### ISSUE TITLE
SSH keys not being saved to /etc/dokploy/ssh

### ISSUE THREAD

**Issue by Yassine-Latreche** [Dokploy/dokploy#565]
### To Reproduce

1. Create a ssh key from dashboard settings
2. try to clone a repo from gitlab/github using the ssh key

### Current vs. Expected behavior

---

### Provide environment information

```bash
---
```

### Which area(s) are affected? (Select all that apply)

Application

### Additional context

the logs shows that the key doesn't exist:
```
Error on  dockerfile / git Error: Cloning into '/etc/dokploy/applications/sdl-project-439be3/code'...
Warning: Identity file /etc/dokploy/ssh/o8JUEY2e9pRVySnmkMim9_rsa not accessible: No such file or directory.
git@gitlab.com: Permission denied (publickey).
fatal: Could not read from remote repository.
```
but when I check the api enpoint `/api/sshKey.all` or the ssh keys section in the setttings page, I get my key.

For the mean time, I created the file manually in my /etc/dokploy/ssh/ and used chmod 600 and it worked

**Comment by Siumauricio** [comment]
Already fixed in #569

**Comment by terrytz** [comment]
Still not fixed with Version: v0.11.2

**Comment by Yassine-Latreche** [comment]
TBH I didn't test it after @Siumauricio said:

> Already fixed in #564

I manually copied the key and moved on.

**Comment by Siumauricio** [comment]
You need to recreate the keys @Yassine-Latreche
